jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/ResourceListing.java

```diff
@@ -145,7 +147,7 @@ public static String getAsHTML(Resource resource, String base, boolean parent, S
             }
         }
 
-        buf.append("<tr><th class=\"name\"><a href=\"?C=N&O=").append(order).append("\">");
+        buf.append("<tr><th class=\"name\"><a href=\"?C=N&amp;O=").append(order).append("\">");
```

Comment:
This would be a legit use of naked `&`, it should not be `&amp;` since it's a query parameter separator.

Reply:
Doh! Leave it as `&amp;` then.